.github/PULL_REQUEST_TEMPLATE/bugfix.md

```diff
@@ -0,0 +1,16 @@
+# Bug fix
+
+## Description
+<!-- Please briefly describe the bug. -->
+
+## Fix
+<!-- Please provide a succinct overview of the fix. -->
+
+# Checklist
+- [ ] Entry added to `CHANGELOG.md`
+- [ ] Regression test added; or
+- [ ] Description of manual testing performed and explanation is included in the code and/or PR.
+- [ ] Added to the correct milestone.
```

Comment:
you could mark this as a "maintainer only" task, since outside contributors can modify the milestone